plugins/foreman_azure/2.x/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: plugin
+title: Foreman AzureRM 2.x Manual
+version: 2.x
+# uncomment for older versions than stable or nightly
+#robots: noindex
+---
+
+# 1. Basics
+
+Foreman AzureRM is the plugin for Azure v2 i.e. Azure Resource Manager API. It allows you to provision and manage your hosts in Azure Resource Manager cloud environment. Users do have restricted access within the Azure subscription (such that only they can create hosts) based on Resource Groups in AzureRM.
+
+This plugin 2.x series has no dependency of foreman_azure 1.x as they refer to different Azure APIs. This 2.x should be compatible with Foreman 1.17 onwards but, it recommended to use latest foreman version.
+
+# 2. Installation
+
+See [the Foreman manual]({{site.baseurl}}plugins/#2.2Packageinstallation).
+
+### Red Hat, CentOS, Scientific Linux (rpm)
+
+Set up the repo as explained in the link above, then run
+
+    # yum install tfm-rubygem-foreman_azure_rm
+
+### Fedora (rpm)
+
+Set up the repo as explained in the link above, then run
+
+    # yum install rubygem-foreman_azure_rm
+
+### Bundle (gem)
+
+Add the following to bundler.d/Gemfile.local.rb in your Foreman installation directory (/usr/share/foreman by default)
+
+    $ gem 'foreman_azure_rm'
+
+Then run `bundle install` from the same directory
+
+# 3. Configuration
+
+Please refer to our documentation [Github](https://github.com/theforeman/foreman_azure_rm#foreman-azurerm-plugin)
+
+# 4. Usage
+
+Once the compute resource with AzureRM provider is created, next step is Host creation. You have to select your AzureRM compute resource on the Hosts page and the relevant fields shall appear on the Virtual Machine tab. You can now fill in your resource group, size of the VM you want to provision, username and password or sshkey with which the user can login to the provisioned machine. In addition to that, you can provide any custom script or command to be executed during the VM creation process. In the Operating Systems tab, you should specify the Architecture, OS, Root Password and the relevant Image that you have created under the compute resource. Last specification is for the Network Interface. You should select a domain defined under Foreman and then select Azure subnet and Public or Private IP format. Upon clicking submit, the host will be successfully provisioned and VM details will then be available.
```

Comment:
Probably worth adding a couple of sentences about how to define a new compute resource, especially the fact that there should be created a resource per region.